static/src/stylesheets/module/experiments/_reading-time.scss

```diff
@@ -0,0 +1,141 @@
+#reading-time.fc-container--thrasher {
```

Comment:
Generally we only want to use class selectors for styling, to avoid extra complexity around specificity: https://developer.mozilla.org/en/docs/Web/CSS/Specificity